lmdeploy/pytorch/nn/linear.py

```diff
@@ -663,7 +664,9 @@ def _weight_loader_tp_rowwise(self, param: torch.nn.Parameter,
                                   loaded_weight: torch.Tensor, rank: int,
                                   world_size: int):
         """weight loader for rowwise linear."""
-        if loaded_weight.dim() == 2 and param.dtype == torch.int8:
+        if loaded_weight.dim() == 2 and param.dtype in (torch.int8,
+                                                        torch.float8_e4m3fn,
```

Comment:
What's the minimum requirement of torch given "torch.float8_e4m3fn" and "torch.float8_e5m2"?

Reply:
torch>=2.3.0 https://pytorch.org/docs/2.3/tensors.html#data-types